utils/mail.go

```diff
@@ -128,14 +129,14 @@ func ConnectToSMTPServer(config *model.Config) (net.Conn, *model.AppError) {
 func NewSMTPClientAdvanced(conn net.Conn, hostname string, connectionInfo *SmtpConnectionInfo) (*smtp.Client, *model.AppError) {
 	c, err := smtp.NewClient(conn, connectionInfo.SmtpServerName+":"+connectionInfo.SmtpPort)
 	if err != nil {
-		l4g.Error(T("utils.mail.new_client.open.error"), err)
```

Comment:
Probably writing a script to remove them.

Reply:
Ok, here's a ticket for that: https://mattermost.atlassian.net/browse/MM-10389. If you want to take it off the books you can delete it or else it can get assigned to whatever team.